etc/sds_operations_gluster-3.8.3.yaml

```diff
@@ -1,160 +1,543 @@
+tendrl_schema_version: 0.1
+
 valid_objects:
+- tendrl_context
 - volume
 - brick
 - peer
-- snapshot
 
 object_details:
+  tendrl_context:
+    enabled: True
+    value: /clusters/$cluster.id/tendrl_context
+    attrs:
+      sds_type:
+        type: String
+        help: "gluster-3.8.3"
   volume:
+    enabled: True
+    value: "/clusters/$cluster.id/volumes/$volume.id"
     attrs:
       volname:
         type: String
+        help: "Name of gluster volume"
       stripe_count:
         type: Integer
+        help: "Stripe count of volume"
       replica_count:
         type: Integer
+        help: "Replica count of volume"
       disperse_count:
         type: Integer
+        help: "Disperse count of volume"
       redundancy_count:
         type: Integer
+        help: "Redundancy count of volume"
       transport:
         type: String
+        help: "Transport type for volume"
       brickdetails:
-        type: list[brick]
+        type: brick[]
+        help: "List of bricks for volume"
       force:
         type: Boolean
-      directory:
-        type: String
-      bitrot:
-        type: Boolean
-      scrub-frequency:
-        type: Integer
-      scrub-throttle:
-        type: Integer
+        help: "If force execute the action"
 
     atoms:
-        info:
-          type: "get"
         create:
-          run: "gluster_bridge.atoms.volume.create"
+          enabled: True
+          uuid: "242f6190-9b37-11e6-950d-a24fc0d9649c"
+          flows: ["CreateVolume"]
+          run: "tendrl.gluster_bridge.atoms.volume.create.Create"
           type: "create"
-          conditions:
-            pre_condition : ["atoms.<atom_name>.<condition_function>1"...]
-            post_condition: ["atoms.<atom_name>.<condition_function>1"...]
-          name: Meaningful name for this atom, should be something that is distinguishable and understandable for notification, debugging, storing and any other similar purposes
+          pre_run : []
+          post_run: []
+          name: "create_volume"
           outputs:
-          - A set, string or list of items that this will be providing (or could provide) to others, used to correlate and associate the thing/s this atom produces, if it produces anything at all.
-
-          global_inputs:
-          - An immutable input_name => value dictionary which specifies any initial inputs that should be automatically injected into the atoms scope before the atom execution commences (this allows for providing atom local values that do not need to be provided by other atoms/dependents).
-          local_inputs:
-          - A set or list of required inputs for this atom’s execute method
-          version: An immutable version that associates version information with this atom. It can be useful in resuming older versions of atoms. Standard major, minor versioning concepts should apply
+            - "volname"
+          inputs:
+            - "volname"
+            - ["stripe_count"]
```

Comment:
Its just a list for "inputs" and for "output" you need to use the structure described here 
https://github.com/Tendrl/bridge_common/blob/master/etc/samples/tendrl_definitions_generic-0.1.sample.yaml#L58